massgov/openmass
# DP-23093: Update org report on flexible org feature branch

**Description:**
Updated the Content Report for Organization Pages to show new section counts from the organization sections field.

**Jira:** (Skip unless you are MA staff)
DP-23093


**To Test:**
- Login and go to /node/add/org_page.
- Verify this text is included in the help text for the “Primary Contact Information” field: Contact information will only appear on your Org page if you add a Contact and Logo section under the "Content" tab.
- Click on the “Overview” tab.
- Verify this text is included in the help text for the “Social Links” field: Social information will only appear on your Org page if you add a Social Media section under the "Content" tab.
- Go to /admin/ma-dash/reports/organization-pages.
- Verify the report matches Prod.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
